docs/data_collection_fundamentals/computer_preparation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Computer preparation
+description: Set up your computer to be able to code scrapers with Node.js and JavaScript. Download Node.js and NPM and run a Hello World script.
+menuWeight: 20.4
+paths:
+    - data-collection-fundamentals/computer-preparation
+---
+
+# [](#prepare) Prepare your computer for programming
+
+Before you can start writing scraper code, you need to have your computer set up for it. In this chapter, we will show you all the tools you need to install to successfully write your first scraper.
+
+## [](#install-node) Install Node.js
+
+Let's start with installation of Node.js. Node.js is an engine for running JavaScript, quite similar to the browser console we used in the previous chapters. You feed it JavaScript code, and it executes it for you. Why not just use the browser console? Simply put, because it's limited in its capabilities. Node.js is way more powerful and is much better suited for coding scrapers.
+
+To install Node.js <a href="https://nodejs.org/en/download/" target="_blank">visit the official Node.js website</a>, download the installer, and follow the instructions.
+
+## [](#install-editor) Install a text editor
+
+There are many text editors you can choose from for programming. You might already have a preferred one so feel free to use that. Just make sure it has syntax highlighting and support for Node.js. If you don't have any text editor, we suggest starting with VSCode. It's free, very popular, and well maintained. <a href="https://code.visualstudio.com/download" target="_blank">Download it here</a>.
+
+## [](#hello-world) Hello World
+
+Before we start, let's confirm that Node.js was successfully installed on your computer. To do that, run those two commands in your terminal and see if they correctly print your Node.js and NPM versions.
+
+```shell
+node -v
+npm -v
+```
+
+> If you're wondering what a "terminal" is, we suggest googling for a terminal tutorial for your operating system because there are many different terminals.
```

Comment:
Would add a note that you can open integrated terminal in VS code for smoother workflow